content/en/tracing/trace_collection/dd_libraries/java.md

```diff
@@ -137,6 +102,39 @@ For other environments, please refer to the [Integrations][5] documentation for
 {{% /tab %}}
 {{< /tabs >}}
 
+### Instrument Your Application
+
+Once the agent is installed, follow the instructions below to begin tracing your applications:
+
+1. Download `dd-java-agent.jar` that contains the latest Agent class files:
+
+   ```shell
+   wget -O dd-java-agent.jar https://dtdg.co/latest-java-tracer
+   ```
+   To access a specific tracer version, visit Datadog's [Maven repository][4].
+
+2. To run your app from an IDE, Maven or Gradle application script, or `java -jar` command, with the Continuous Profiler, deployment tracking, logs injection (if you are sending logs to Datadog), and trace volume control, add the `-javaagent` JVM argument and the following configuration options, as applicable:
+
+    ```text
+    java -javaagent:/path/to/dd-java-agent.jar -Ddd.profiling.enabled=true -XX:FlightRecorderOptions=stackdepth=256 -Ddd.logs.injection=true -Ddd.service=my-app -Ddd.env=staging -jar path/to/your/app.jar -Ddd.version=1.0
+    ```
+
+    **Note:** Enabling profiling may impact your bill depending on your APM bundle. See the [pricing page][5] for more information.
+
+| Environment Variable      | System Property                     | Description|
+| --------- | --------------------------------- | ------------ |
+| `DD_ENV`      | `dd.env`                  | Your application environment (`production`, `staging`, etc.) |
+| `DD_SERVICE`   | `dd.service`     | The name of a set of processes that do the same job. Used for grouping stats for your application. |
+| `DD_VERSION` | `dd.version` |  Your application version (for example, `2.5`, `202003181415`, `1.3-alpha`, etc.) |
+| `DD_PROFILING_ENABLED`      | `dd.profiling.enabled`          | Enable the [Continous Profiler][6] |
+| `DD_LOGS_INJECTION`   | `dd.logs.injection`     | Enable automatic MDC key injection for Datadog trace and span IDs. See [Advanced Usage][7] for details. |
+| `DD_TRACE_SAMPLE_RATE` | `dd.trace.sample.rate` |   Set a sampling rate at the root of the trace for all services.     |
+| `DD_TRACE_SAMPLING_SERVICE_RULES` | `dd.trace.sampling.service.rules` |   Set a sampling rate at the root of the trace for services that match the specified rule.    |
+
+Additional [configuration options](#configuration) are described below.
+
+3. Ensure the Datadog Agent is configured for APM and reachable from your application from the environment specific instructions [below](#configure-the-datadog-agent-for-apm).
```

Comment:
We've now moved these instructions above, so maybe we can just assume the person has done them, and delete item 3.